topics/qt/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
+created_by: Haavard Nord, Eirik Chambe-Eng
+display_name: Qt
+github_url: https://github.com/qt
+logo: Qt_logo.png
+released: 20 May 1995
+short_description: Qt is a cross-platform application framework that is used for developing application software.
+topic: qt
```

Comment:
I see qt is on the list https://github.com/github/explore/blob/master/topics-todo.md. Would you mind editing topics-todo.md in this branch to check off `qt`? You can replace the line `[ ] qt` with `[x] qt` to do so.